benchmark/poseidon2/poseidon2_config.h

```diff
@@ -6,37 +6,28 @@
 #include <string>
 #include <vector>
 
+#include "benchmark/config.h"
+
 namespace tachyon {
```

Comment:
```suggestion
namespace tachyon::benchmark {
```

Reply:
I will add the namespace in separate commit.